FontoXPathCSharp/CompileAstToExpression.cs

```diff
@@ -475,9 +474,9 @@ or AstNodeName.WindowClause
             );
 
             returnExpr = new ForExpression<TNode>(
-                forClauseItem.FollowPath(AstNodeName.TypedVariableBinding, AstNodeName.VarName)?.GetQName(),
+                forClauseItem.FollowPath(AstNodeName.TypedVariableBinding, AstNodeName.VarName)?.GetQName()!,
                 CompileAst(expression, DisallowUpdating(options)),
```

Comment:
Possible null reference argument for parameter 'ast' in 'AbstractExpression<TNode> CompileAstToExpression<TNode>.CompileAst(Ast ast, CompilationOptions options)'.